### ISSUE TITLE
For getDocuments **attributesToRetrieve** param works only if is string type.

### ISSUE THREAD

**Issue by vitalijm** [meilisearch/MeiliSearch#2105]
**Describe the bug**
For getDocuments **attributesToRetrieve** param works only if is string type.
If array type, then throwing error:
`Fatal error: Uncaught MeiliSearch ApiException: Http Status: 400 - Message: Query deserialize error: unknown field `attributesToRetrieve[0]`, expected one of `offset`, `limit`, `attributesToRetrieve` - Code: bad_request - Type: invalid_request - Link: https://docs.meilisearch.com/errors#bad_request thrown in /app/vendor/meilisearch/meilisearch-php/src/Http/Client.php on line 215`

**To Reproduce**
1.
`$hits = $client->index('movies')->getDocuments(['limit' => 2, 'offset' => '2', '' => ['title','poster']]);`
2. See error

**MeiliSearch version:** [e.g. v0.25.0]

**Additional context**
OS: debian 11, via docker

**Comment by MarinPostma** [comment]
uhmm looks like a meilisearch-php issue, wdyt @meilisearch/integration-admin-team ?

**Comment by curquiza** [comment]
Indeed, looks like the query parameters are wrongly sent to the search engine! Moving this to meilisearch-php
